R/simulation.R

```diff
@@ -21,17 +21,11 @@ Simulation <- R6::R6Class(
     path = function(value) {
       private$readOnlyProperty("path", value, self$root$path)
     },
-    #' @field settings SimulationSettings object for the simulation (read-only)
-    settings = function(value) {
-      if (missing(value)) {
-        private$.settings
-      } else {
-        private$throwPropertyIsReadonly("settings")
-      }
-    },
     #' @field solver SimulationSolver object for the simulation (read-only)
     solver = function(value) {
-      private$readOnlyProperty("solver", value, private$.settings$solver)

```

Comment:
This was the issue. Not sure why but eh... it seems that doing a$b$c is trying to reassign b . weird

Reply:
R...